vignettes/climr_workflow_beg.Rmd

```diff
@@ -40,64 +40,89 @@ cpath <- if (interactive()) {
 options("climr.cache.path" = cpath)
 ```
 
+###Using the `climr_downscale()` function
 
-The most basic use of `climr` is to pass a `data.table` of point coordinates, IDs
-and elevation to `climr_downscale()` and select the type of climate projection or 
-values wanted - e.g. below we ask for future projections in three periods, using
-two emissions scenarios and two General Circulation Models. We also specify we want
-3 individual runs of each model/scenario combination, in addition to the ensemble mean
-(returned by default).
+The basic use of `climr` is to pass a `data.table` of point coordinates,
+IDs and elevation to `climr_downscale()` and select the type of climate
+projection or values wanted - e.g. below we ask for future projections
+in two periods, using two emissions scenarios and two General
+Circulation Models. We also specify we want 3 individual runs of each
+model/scenario combination, in addition to the ensemble mean (returned
+by default).
 
 ```{r}
 library(climr)
 
-## A data.table of point coordinates, IDs and elevation
-data("xyzDT")
+## provide a data.frame or data.table of point coordinates, IDs and elevation
+my_points <- data.frame(
```

Comment:
I'm totally fine with this, but is there a reason why you prefer not using package data - so users can "see" the data structure? If this is the case, we should see whether `data(xyzDT)` is used elsewhere and if not, remove it from the package

Reply:
the problem with data(xyzDT) is that it covers a big area and so it takes a long time to cache the data for first use. not a great introduction to climr. we could create a different xyzDT over a smaller area.